akka-docs/src/main/paradox/project/links.md

```diff
@@ -8,8 +8,8 @@ Akka is part of the [Akka Platform](https://www.lightbend.com/akka-platform).
 ## Sponsors
 
 **Lightbend** is the company behind the Akka Project, Scala Programming Language,
-Play Web Framework, Lagom, sbt and many other open source projects. 
-It also provides the Lightbend Reactive Platform, which is powered by an open source core and commercial Enterprise Suite for building scalable Reactive systems on the JVM. Learn more at [lightbend.com](https://www.lightbend.com).
+Play Web Framework, Lagom, sbt and many other open source and source available projects. 
```

Comment:
Play is currently Community Maintained?

Reply:
Yes, this whole section would need some rewrite and that is a later task. Lightbend is still the company **behind** Play, which is what the sentence states.